packages/dds/matrix/src/test/memory/matrix.spec.ts

```diff
@@ -52,7 +52,7 @@ describe("Matrix memory usage", () => {
 		);
 
 		const numbersOfEntriesForTests = isInPerformanceTestingMode
-			? [1000, 10_000, 100_000]
+			? [100, 1000, 10_000]
```

Comment:
Minor: I looked through our codebase and there's no consensus on whether to write `1_000` or `1000`. I propose 1_000 for consistency (e.g., the `_` will always be after thousands/millions/etc.), but I'm not sure how much work it would be to clean up the other places we use separators in test code.

Reply:
I would agree but I think some of our linting/formatting rules don't? @Josmithr might remember.